etc/org.opencastproject.ingest.scanner.InboxScannerService-inbox.cfg

```diff
@@ -3,7 +3,7 @@ user.name=admin
 user.organization=mh_default_org
 
 # Specify the workflow definition (by its identifier) to run for media, ingested from the inbox
-workflow.definition=schedule-and-upload
+workflow.definition=fast
```

Comment:
Do you really want to set the fast testing workflow as default?

Reply:
I reverted this change